subgraphs/lido/protocols/lido/config/templates/lido.template.yaml

```diff
@@ -1,6 +1,11 @@
 specVersion: 0.0.4
 schema:
   file: ./schema.graphql
+features:
+  - grafting
+graft:
+  base: Qme9Vb2FTRRKaoR25bDbtL6JzTcmuQfTgbfmoQXigUynNd
+  block: 17172465
```

Comment:
Can you either remove this from the template or add it to the config so we can turn it on/off?